pages/dashboard.js

```diff
@@ -1,5 +1,5 @@
 import { useEffect, useState } from "react";
-import { Alert, Button, Space, Typography, Col, Row } from "antd";
+import { Alert, Button, Space, Typography, Col, notification, Row } from "antd";
```

Comment:
Why do we need to import `notification` here? (Same for the other files)

Reply:
You don't. I added it for testing and forgot to remove it 😅